src/vcu/inc/controls.h

```diff
@@ -0,0 +1,71 @@
+#ifndef __TORQUE_CALC
+#define __TORQUE_CALC
+
+#include "stm32f2xx_hal.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "can_handles.h"
+#include "fault_pedalbox.h"
+
+// TODO: Extract MAX_TORQUE from RMS EEPROM (over CAN?)
+#define MAX_TORQUE    2400 // unit: dNm
+#define MAX_ACCEL_VAL 1000
+#define MIN_ACCEL_VAL 0
+
+// Launch control contants
+// After wheel speed crosses this threshold, start using slip controller
+#define LC_WS_THRESH 45000
+#define LC_cGR 347 // Gear ratio times 100
+#define LC_ACCEL_BEGIN 950 // 95%
+#define LC_ACCEL_RELEASE 50 // 5%
+#define LC_BRAKE_BEGIN 100 // We want a lower threshold
+
+// RG = regen
+#define RG_MOTOR_SPEED_THRESH 250            // RPM
```

Comment:
It is good to use MIN or MAX in lieu of THRESH in order to indicate the type of bound intended.